impl/src/main/java/org/ehcache/impl/internal/store/disk/OffHeapDiskStore.java

```diff
@@ -466,6 +463,7 @@ public void initAuthoritativeTier(AuthoritativeTier<?, ?> resource) {
    * This is kind of a hack, but it's safe to use this if the regular portability
    * is stateless.
    */
+  @SuppressWarnings("unchecked")
```

Comment:
I prefer having the annotation exist where it is really needed. I hate the fact that the annotation does not have an attribute to explain why you are using it btw.